code/datums/mood_events/generic_negative_events.dm

```diff
@@ -84,7 +84,7 @@
   mood_change = -3
 
 /datum/mood_event/brain_damage/add_effects()
-  var/damage_message = pick_list_replacements("brain_damage_lines.json", "brain_damage")
+  var/damage_message = pick_list_replacements(BRAIN_DAMAGE_FILE, "brain_damage")
```

Comment:
Neither of these seem right? unless pick_list_replacements doesn't take a list

Reply:
brain_damage_lines.json was renamed to traumas.json, which BRAIN_DAMAGE_FILE represents.